azure/python/python-artifact.yml

```diff
@@ -0,0 +1,71 @@
+
+
+stages:
+- stage: Testing
+  jobs:
+  - job: 'Test'
+    pool:
+      vmImage: 'ubuntu-16.04'
+    strategy:
+      matrix:
+        Python36:
+          python.version: '3.6'
+        Python37:
+          python.version: '3.7'
+        Python38:
+          python.version: '3.8'
+
+    steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(python.version)'
+
+    - script: pip install flit
+      displayName: 'Install flit build tool'
+
+    - script: flit install --symlink
+      displayName: 'Install library and its dependencies'
+
+    - script: pytest
+      displayName: 'Test with pytest'
+
+- stage: "Publish"
+  condition: and(succeeded(), eq(variables['release.version'], ''), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Shouldn't the release version non-empty?

Reply:
you are right!